PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx

```diff
@@ -864,7 +860,21 @@ void AliAnalysisTaskMaterialHistos::UserExec(Option_t *){
     hBCNumber[iCut]->Fill(fBCNumber);
   }
 
-  if(fDoSelectBCNumber) DoSelectBCNumbers();
+  if(fDoSelectBCNumber){
+    if(fIsMC == 0 ){
+      fRunNumber = fInputEvent->GetRunNumber();
+      fBCNumber  = fInputEvent->GetBunchCrossNumber();
+      if(fRunNumber >= 265332 && fRunNumber <= 265344){//Fill 5506
+	if(1390 > fBCNumber || fBCNumber > 1540){
+	return;
+	}
+      }else if(fRunNumber >= 265377 && fRunNumber <= 265388){//Fill 5507
+	if(3010 > fBCNumber || fBCNumber > 3140){
+	  return;
+	}
+      }
+    }
+  }
```

Comment:
Maybe you want to put this part a bit later in UserExec in order to fill a column in the NEvents to keep track of how many events were rejected. (Maybe this might also be necessary for some normalization in the end.)